### PR TITLE
gitea: update to 1.16.1

### PR DESCRIPTION
**Pull request by zen0bit**
PS: Already released 1.16.1 but not for download src version...

#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64 libc

### CODE REVIEW

**Comment by paper42** [comment]
Please fix the commit message according to the conventions mentioned in CONTRIBUTING.md. I can see a -src archive in the 1.16.1 release, could you explain why it's not possible to update?

**Comment by zen0bit** [comment]
idk Can't download but now its ok
Building 1.16.1...

Is it possible change commit message here on github?
(Working through browser now not on cli...)

**Comment by paper42** [comment]
> idk Can't download but now its ok Building 1.16.1...

Maybe they added it to the release if that's possible.

> Is it possible change commit message here on github? (Working through browser now not on cli...)

I don't know, but I know that people who use github's UI often mess up PRs and are not able to fix them. I would recommend just using git cli.

**Comment by zen0bit** [comment]
I think I will learn it eventually :+1: 
Don't kill me yet

File: srcpkgs/gitea/template [hunk 1]
```diff
@@ -31,8 +31,8 @@ maintainer="fosslinux <fosslinux@aussies.space>"
 license="MIT"
 homepage="https://gitea.io"
 changelog="https://github.com/go-gitea/gitea/blob/master/CHANGELOG.md"
-distfiles="https://github.com/go-gitea/gitea/releases/download/v${version}/gitea-src-${version}.tar.gz"
-checksum=a05745e5c1b5e9b51caa3d96fd93689176689865a0c249aa477d411e5c082045
+distfiles="https://github.com/go-gitea/gitea/releases/download/v1.16.1/gitea-src-1.16.1.tar.gz"
```
Comment (paper42) [inline srcpkgs/gitea/template:34]: use ${version} here, otherwise lgtm

Comment (zen0bit) [reply srcpkgs/gitea/template:34]: i every time misss something...
sry

**Comment by paper42** [comment]
now it's 2 commits, but it should be one, please squash them

**Comment by zen0bit** [comment]
#35466
PS:sry but easier from gihub UI
This can be deleted...

**Comment by abenson** [comment]
Please don't open/close PRs.  It just adds noise.  I also don't recommend doing this through the web UI, it's easier interacting with git directly.

**Comment by zen0bit** [comment]
i know, true...
I am slowly getting into..

**Pull request closed without merging**